src/main/app/src/app/fout-afhandeling/fout-afhandeling.service.ts

```diff
@@ -119,12 +119,17 @@ export class FoutAfhandelingService {
 
   public httpErrorAfhandelen(err: HttpErrorResponse): Observable<never> {
     if (err.status === 400) {
-      return this.openFoutDialog(this.translate.instant(err.error.message));
+      return this.openFoutDialog(
+        this.translate.instant(
+          err.error.message || err.message || "dialoog.body.error.technisch",
+        ),
+      );
     }
+
     this.foutmelding = err.message;
     if (err.error instanceof ErrorEvent) {
       // client-side error
-      this.foutmelding = `Er is een fout opgetreden`;
+      this.foutmelding = this.translate.instant(`dialoog.body.error.fout`);
```

Comment:
```suggestion
      this.foutmelding = this.translate.instant("dialoog.body.error.fout");
```
or why use backticks here?

Reply:
no idea; just put some other text in between; but fixed